src/App.js

```diff
@@ -2,18 +2,62 @@ import React from 'react';
 import './App.css';
 
 import users from './api/users';
+import todos from './api/todos';
+import NewToDo from './NewToDo/NewToDo';
+import ToDoList from './ToDoList/ToDoList';
 
-function App() {
-  return (
-    <div className="App">
-      <h1>Add todo form</h1>
-
-      <p>
-        <span>Users: </span>
-        {users.length}
-      </p>
-    </div>
-  );
+const newToDo = todos.map(todo => (
```

Comment:
```suggestion
const todoList = todos.map(todo => (
```

this is not one element

Reply:
not fixed